_posts/2019-03-01-custom-editorial-features.markdown

```diff
@@ -0,0 +1,94 @@
+- The Year in Visual Culture 2018 - [Components](https://github.com/artsy/reaction/blob/master/src/Components/Publishing/EditorialFeature/Components/Eoy2018Culture.tsx) | [Article](https://www.artsy.net/article/artsy-editorial-people-defined-visual-culture-2018)
+
+# Custom articles by domain:
+
+**1. [In Force (Artsy.net)](https://github.com/artsy/force)**
+
+- Whether an article requires a custom layout is determined in Force’s [article routing](https://github.com/artsy/force/blob/master/src/desktop/apps/article/routes.ts#L140). This is achieved by passing the prop `customEditorial`-- a string shorthand for a specific article-- to Reaction’s top-level `Article` component. The `customEditorial` prop is pulled from Force’s editorial feature "[master list](https://github.com/artsy/force/blob/master/src/desktop/apps/article/editorial_features.ts#L13)", which ties an `article._id` to a communicative string that will be received by Reaction. In addition to data saved to an article model, the component will also receive all data displayed in the footer including related articles and display ads. Custom articles are rendered as a standalone page, meaning they are excluded from infinite scroll and do not render the main site header.
+
+**2. [In Reaction (Artsy’s component library)](https://github.com/artsy/reaction)**
+
+- In Reaction’s top-level [`Article`](https://github.com/artsy/reaction/blob/master/src/Components/Publishing/Article.tsx#L64) component, the presence of a `customEditoral` prop routes an article to the [`ArticleWithFullScreen`](https://github.com/artsy/reaction/blob/master/src/Components/Publishing/Layouts/ArticleWithFullScreen.tsx) component. From here, the article is given context for image slideshows and tooltip helpers, and the  `EditorialFeature` component is rendered rather than the component designated by the article’s specified layout. A `FeatureLayout` is displayed by default, but any article can be converted into a custom feature, regardless of the `article.layout` value. Inside the `EditorialFeature` component, a switch statement is used to associate the string variable for the feature with its affiliated top-level component.
+
+**3. [In Writer/Positron (CMS & API for articles)](https://github.com/artsy/positron)**
+
+- Because `EditorialFeature` accepts an article data-model, it can be edited using the Writer CMS. However it is important to note that a custom layout is rendered by Force only. While editing, what users see is dicated by the `article.layout` property. Writer's features are exposed based on this property, so a particular custom article’s layout should be determined by the features most suited to the content and design.  For example, if you need a header-image or video, a feature article would be a logical choice because that content can easily be created and edited in Writer. If the article relies heavily on content from related articles, you might choose to customise a series article instead.
```

Comment:
You've used the British spelling for "customise" here, and the US spelling for "customization" above. It may be good to standardize (standardise?) those.

Reply:
good catch, my spellchecker has been confused since my last trip to EU 🙃